app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
```

Comment:
Question: should this be `:user`, or does it need to be `:users` since there are multiple players?